### ISSUE TITLE
Video sent from ff is not playable.

### ISSUE THREAD

**Issue by 532910** [vector-im/riot-web#14330]
While a video sent from riot-android has playback controls in all riots:
![image](https://user-images.githubusercontent.com/14617699/86512683-e2ab8d00-be0c-11ea-8ff0-d807867ca6f4.png)

The same video sent from riot-web in ff isn't playable:
![image](https://user-images.githubusercontent.com/14617699/86512712-138bc200-be0d-11ea-8ee7-187bd3783107.png)
![image](https://user-images.githubusercontent.com/14617699/86512726-2a321900-be0d-11ea-82be-77a9f7efb726.png)

**Comment by t3chguy** [comment]
Does your firefox have anti-fingerprinting enabled? Anything which breaks canvases can causes the file to be mis-detected as not being a video.

**Comment by 532910** [comment]
I don't know, but ff with a new clean profile works the same.

**Comment by 532910** [comment]
ff 78.0.1 but I've seen this issue before

**Comment by 532910** [comment]
BTW, image files are shown correctly sent from the same ff.

**Comment by 532910** [comment]
The same as #14328, this issue is HEVC related.

**Comment by t3chguy** [comment]
https://caniuse.com/#feat=hevc

**Comment by 532910** [comment]
The last screen is from android. While it can play HEVC, the video has no controls.

**Comment by t3chguy** [comment]
Right, the issue is if the sending browser doesn't support HEVC then it can't verify that the file is a video so it sends it as `m.file` and not `m.video`

**Comment by ara4n** [comment]
see https://github.com/element-hq/element-web/issues/10146
